triagebot.toml

```diff
@@ -1208,7 +1212,7 @@ project-exploit-mitigations = [
 "/src/doc/nomicon" =                                     ["@ehuss"]
 "/src/doc/reference" =                                   ["@ehuss"]
 "/src/doc/rust-by-example" =                             ["@ehuss"]
-"/src/doc/rustc-dev-guide" =                             ["@kobzol", "@jieyouxu"]
+"/src/doc/rustc-dev-guide" =                             ["@kobzol", "@jieyouxu", "@BoxyUwU"]
```

Comment:
I also added myself here but I wonder if we should actually be just assigning compiler since in most cases it will be compiler PRs that happen to also touch the dev guide

Reply:
I don't think we should ping the entire compiler team on every change to the dev guide, I personally do not want to be pinged

Reply:
oh wait you mean assignment. yeah assigning compiler makes sense

Reply:
I'm actually just going to go ahead and do that, let me know what you think. With the ping group we'll all get notified whenever anyone touches the subtree *anyway* so I don't know if there's a point in explicitly maintaining a separate list here from compiler when these PRs are going to want a compiler reviewer anyway :woman_shrugging: